src/components/CityHeader/CityHeader.scss

```diff
@@ -33,62 +33,44 @@
   @include for-tablet-landscape-up {
     padding: 32px;
   }
-
-  @include for-desktop-up {
-    padding: 46px 32px;
-  }
 }
 
-.backgroundPanelWrapper {
+.photoWrapper {
   position: absolute;
-  display: flex;
-  flex-basis: 0;
-  align-items: stretch;
   left: 0;
   right: 0;
   top: 0;
   bottom: 0;
+  overflow: hidden;
 }
 
-.backgroundPanel {
-  flex: auto;
-
-  @include for-tablet-portrait-up {
-    &:last-child {
-      flex: 2;
-    }
+.photo {
+  height: 100%;
+  width: auto;
+  display: block;
+  margin: 0 auto;
+
+  // For browsers that support object-fit (and @supports)
+  // set the width to 100%
+  @supports (object-fit: cover) {
+    object-fit: cover;
+    width: 100%;
   }
```

Comment:
If ever you're wondering which browsers support some feature, there's https://caniuse.com/#feat=css-featurequeries (that's which browsers support the supports feature).

The browser support table on MDN is not always right.